portal-impl/src/com/liferay/portal/service.xml

```diff
@@ -1762,6 +1762,40 @@
 			<finder-column name="smallValue" />
 		</finder>
 	</entity>
+	<entity change-tracking-enabled="true" local-service="true" name="PropagationPermission" remote-service="true">
```

Comment:
The table name suggested by the product team is`PermissionPropagationEntry`?

Reply:
I've updated the code.
Thank you!